engine/classes/Elgg/Database/Seeds/Seed.php

```diff
@@ -21,17 +21,17 @@ abstract class Seed implements Seedable {
 	/**
 	 * @var int Max number of items to be created by the seed
 	 */
-	protected $limit = 20;
+	protected int $limit = 20;
```

Comment:
you can also set this value to the defaultLimit function in the constructor

Reply:
moved to constructor